corehq/apps/export/utils.py

```diff
@@ -250,6 +253,19 @@ def _convert_transform(serializable_transform):
     return None
 
 
+def _get_for_single_node_repeat(tables, column_path, transform):
+    for new_table in tables:
+        column_dot_path = '.'.join(map(lambda node: node.name, column_path))
```

Comment:
seems like this could be done outside the loop which might be more readable and more efficient